scripts/postinstall.js

```diff
@@ -0,0 +1,31 @@
+const chalk = require('chalk')
+const fs = require('fs-extra')
+
+// proccess.env.PWD is undifined on Windows https://github.com/mrblueblue/gettext-loader/issues/18
+const pwd = process.cwd();
+const destDir = `${pwd}/../@types/nexus-plugin-prisma`
+
+async function copyTypes() {
+  try {
+    await fs.ensureDir(destDir)
+    await fs.copy(`${pwd}/global-type.d.ts`, `${destDir}/index.d.ts`)
+  } catch (err) {
+    console.error(err)
```

Comment:
```suggestion
    console.error(err)
    process.exit(1)
```

Does what we were achieving with `&&` IIUC